standards/core/rendezvous.md

```diff
@@ -0,0 +1,64 @@
+---
+title: WAKU-RENDEZVOUS
+name: Waku rendez vous discovery
```

Comment:
```suggestion
name: Waku Rendezvous Discovery
```

Reply:
I hate it but I guess it's the English language...

Reply:
😆 I apologise on behalf of English.